casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -288,19 +287,19 @@ class RuntimeManagerImpl[F[_]: Concurrent: Metrics: Span: Log](
       startHash: StateHash,
       terms: Seq[InternalProcessedDeploy],
       replayDeploy: InternalProcessedDeploy => F[Option[ReplayFailure]]
-  ): F[Either[ReplayFailure, StateHash]] =
-    (for {
-      _ <- EitherT.right(runtime.replaySpace.reset(Blake2b256Hash.fromByteString(startHash)))
-      _ <- EitherT(
-            terms.tailRecM { ts =>
-              if (ts.isEmpty)
-                ().asRight[ReplayFailure].asRight[Seq[InternalProcessedDeploy]].pure[F]
-              else replayDeploy(ts.head).map(_.map(_.asLeft[Unit]).toRight(ts.tail))
-            }
-          )
-      _   <- EitherT.right(Span[F].mark("before-replay-deploys-create-checkpoint"))
-      res <- EitherT.right[ReplayFailure](runtime.replaySpace.createCheckpoint())
-    } yield res.root.toByteString).value
+  ): F[Either[ReplayFailure, StateHash]] = {
+    import cats.implicits.{catsStdInstancesForList, catsStdInstancesForOption}
```

Comment:
I'd go with the less scary `cats.instances.{list._, option._}` or however scala wants the syntax :)